domehunter/protos/Makefile

```diff
@@ -1,10 +1,10 @@
 # Makefile for libHuntsmanDome
 
 CXX = g++
-CFLAGS = -fPIC -Wall -Wextra -O2 -g -DSB_LINUX_BUILD -I. -I./src/licensedinterfaces/\
+CFLAGS = -fPIC  -O2 -g -DSB_LINUX_BUILD -I. -I./src/licensedinterfaces/\
```

Comment:
I see a LINUX thing here. Maybe need a `Makefile_linux` and `Makefile_osx` as related to this issue: #25